xcube_cmems/cmems.py

```diff
@@ -68,8 +69,8 @@ def __init__(self,
         self.opendap_dataset_ids = {}
 
         if not self.cmems_username or not self.cmems_password:
-            raise Exception('CmemsDataStore needs cmems credentials in env vars'
-                            ' CMEMS_USERNAME and CMEMS_PASSWORD or to be '
+            raise Exception('CmemsDataStore needs cmems credentials in env '
+                            'vars CMEMS_USERNAME and CMEMS_PASSWORD or to be '
                             'provided as store params')
```

Comment:
If they are store params, they must be written in small caps, so you should better write their full names here.